docs/tempo/website/operations/backend_search.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Backend Search
+weight: 9
+---
+
+Backend search can be operationally more complex than traditionally operating Tempo. This is largely due
+to how new it is and how our defaults don't support it well. Search is very much a work in progress and 
+we expect this to improve as we continue to refine this feature. 
```

Comment:
> _This is largely due to how new it is and how our defaults don't support it well._

What defaults are you referring to? From my understanding it's tricky to operate because it's so compute-intensive when searching a large time range. To be able to scale compute we rely on functions, which makes deployment much harder.

Reply:
I'm referring to the settings that are suggested as tunables in this doc. These:

querier.max_concurrent_queries
server.http_server_*_timeout
query_frontend.max_outstanding_per_tenant